tests/_support/AcceptanceTester.php

```diff
@@ -23,4 +23,39 @@ class AcceptanceTester extends \Codeception\Actor
    /**
     * Define custom actions here
     */
+
+   	/**
+	 * Sets tickets enabled post types
+	 *
+	 * @param  array $post_types The post types to be enabled for tickets. Can be empty.
+	 */
+	public function haveTicketablePostTypes ( array $post_types = array( ) ) {
```

Comment:
our method/function/var names should always be `snake_case` in PHP

Reply:
@ggwicz in this case I'm following the casing of the 'standard' of the existing Actor methods and the ones added by WP Browser. Not sure if my approach is the correct one.

Reply:
Both ways are really fine. My personal way is: anything I add in the context of WordPress related work I `snake_case`.